Automattic/wp-calypso
# Podcasting: add indicator to Editor and Category settings

This creates a `PodcastingIndicator` component, and adds it to the `termTreeSelector`, `EditorActionBar`, and `TaxonomyManager`. The indicator signifies if a post is currently assigned the selected Podcasting category, or if a category is currently set as the Podcasting category.

All code is behind the `manage/site-settings/podcasting` feature flag.
See p3Ex-32D-p2. 

**In EditorActionBar w/Tooltip:**
![screen shot 2018-04-07 at 2 48 42 pm](https://user-images.githubusercontent.com/942359/38459064-1dbe7c56-3a73-11e8-9c3f-062a60bb6452.png)

**In TermTreeSelector:**
<img width="255" alt="screen shot 2018-04-07 at 2 51 39 pm" src="https://user-images.githubusercontent.com/942359/38459082-56522950-3a73-11e8-93ba-edb5b27fefac.png">

**In TaxonomyManager:**
<img width="750" alt="screen shot 2018-04-07 at 2 52 18 pm" src="https://user-images.githubusercontent.com/942359/38459088-613137f8-3a73-11e8-946e-6629d437d766.png">

**To test:**
With the `manage/site-settings/podcasting flag` enabled, D10933-code applied to your sandbox, and the API sandboxed:

- On a site with Podcasting enabled, visit the post editor.
- Verify in the Category list, that the podcasting category is marked with the indicator.
- Visit or create a post assigned to the podcasting category.
- Verify that the indicator and tooltip are visible in the action bar at the top of the post editor.
- Deselect the podcasting category for a post and save.
- Verify that the indicator is not visible in the action bar.
- Visit the Taxonomy Manager in Settings > Writing > Categories.
- Verify that the indicator and tooltip are visible beside the selected podcasting category.
- Disable podcasting from the Settings > Writing > Podcasting details screen.
- Verify that the indicator is not visible in any of the previous areas.
